test-lit/semantics/implicit05.f90

```diff
@@ -0,0 +1,7 @@
+! RUN: not %flang -fdebug-resolve-names -fparse-only %s 2>&1 | FileCheck --check-prefixes=ERROR %s
```

Comment:
why is the original file deleted and a new file created?

Reply:
See [here](https://github.com/flang-compiler/f18/pull/934#discussion_r367391638)